### ISSUE TITLE
FR: advPlot string S&R

### ISSUE THREAD

**Issue by DrJKL** [TinyTerra/ComfyUI_tinyterraNodes#154]
Similar to the current string set, but with a target string and a list of replacement values?

**Comment by DrJKL** [comment]
Not sure if there's a simple way to already do this in the current node.

**Comment by TinyTerra** [comment]
Search and Replace:
if you include %search;replace% as the value it will use the current nodes value and do a search and replace using these values.
you can include more than one to replace different strings


It is not functioning as it should, let me know

**Comment by DrJKL** [comment]
Yeah, I think it's working properly, this would be more of a convenience thing.
Here's an example workflow: 
![__00012](https://github.com/user-attachments/assets/c6b78d94-a85a-447b-8fc2-5465340ed313)

I'd just like to not have to repeat the `search` string for each `replace` value.

**Comment by TinyTerra** [comment]
Ah I see,
I've added a replace mode to the string node - setting it to true will show a search_text widget


![image](https://github.com/user-attachments/assets/dc501122-19e2-41a0-9a51-cd055f9cf9bd)
